libr/core/cmd_search.c

```diff
@@ -207,7 +207,7 @@ static const char *help_msg_slash_r[] = {
 };
 
 static const char *help_msg_slash_R[] = {
-	"Usage: /R", "", "search for ROP gadgets",
+	"Usage: /R", "", "search for ROP gadgets (hint: quote the command if your regex uses special chars)",
```

Comment:
optional : (hint:  surround your regex with a double quotation if your regex uses special chars. eg: "regex1 | regex2" ) 

I think it is more clear.

Reply:
thats not correct. because its the whole command that needs to be quoted or just start the command with ""

Reply:
Yes. This was before learning it yesterday. We need the word R2 command with the quote.

should render like this:
```
search for ROP gadgets (hint: quote the whole r2 command if your regex uses special chars. eg: "/R/ regx1;regx2")
```

Reply:
that's too large to fit in a 80x24 screen for something that affects all commands, imho is better to add a hint and put a proper explanation somewhere else in a generic way for all the commands that the user may want to use special foes

Reply:
i mean, all shells have special characters, and none of them reminds the user to escape the chars

Reply:
Fair enough. Would the length of you current changes allow for 2 more chars for (r2) ?

Reply:
Fair enough. Would the length of you current changes allow for 2 more chars for (r2) ?